diff --git a/src/mame/layout/a91du.lay b/src/mame/layout/a91du.lay
new file mode 100644
index 00000000..5c638a11
--- /dev/null
+++ b/src/mame/layout/a91du.lay
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="digit" defstate="0">
+		<led7seg><color red="1.0" green="0.12" blue="0.1" /></led7seg>
+	</element>
+	<element name="panel"><rect><color red="0.16" green="0.16" blue="0.17" /></rect></element>
+	<element name="window"><rect><color red="0.03" green="0.0" blue="0.0" /></rect></element>
+	<element name="name"><text string="ALFASKOP 91" align="1"><color red="0.75" green="0.75" blue="0.75" /></text></element>
+
+	<view name="DU 4110 and A91 front panel">
+		<screen index="0"><bounds x="0" y="0" width="800" height="600" /></screen>
+		<element ref="panel"><bounds x="0" y="600" width="800" height="80" /></element>
+		<element ref="name"><bounds x="24" y="626" width="300" height="28" /></element>
+		<element ref="window"><bounds x="680" y="610" width="100" height="60" /></element>
+		<element name="digit0" ref="digit"><bounds x="692" y="616" width="36" height="48" /></element>
+		<element name="digit1" ref="digit"><bounds x="732" y="616" width="36" height="48" /></element>
+	</view>
+</mamelayout>
